### PR TITLE
removed replace 0 card message

### PR DESCRIPTION
**Pull request by BRHoward**
just removes the possible "removed 0 cards" message if no cards are selected in the replace cards screen

### CODE REVIEW

**Comment by BRHoward** [comment]
this should fix #138

**Comment by JamesAdamHughes** [comment]
Yea seems fine, maybe add a quick test?

File: client/Angular/services/gameService.js [hunk 1]
```diff
@@ -51,9 +51,11 @@ ClonageApp.service('gameService', ['communicationService', 'dynamicTextService',
 
 	//sends off all the cards that the user wants to replace and resets array
 	function replaceCardsSubmit() {
-		sendMessage("GAME replace cards", {
-			cardsToReplace: cardsToReplace
-		});
+		if (cardsToReplace.length > 0) {
```
Comment (JamesAdamHughes) [inline client/Angular/services/gameService.js]: can we disable the button if they haven't selected any cards as well?

**Comment by JamesAdamHughes** [comment]
:neckbeard

**Pull request merged**